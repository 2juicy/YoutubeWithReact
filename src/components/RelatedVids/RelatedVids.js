import React from "react";
import "./RelatedVids.scss";
import VidPreview from "../VidPreview/VidPreview";

export default function RelatedVids() {
  return (
    <div className="related-vids">
      <VidPreview horizontal />
      <VidPreview horizontal />
      <VidPreview horizontal />
      <VidPreview horizontal />
      <VidPreview horizontal />
    </div>
  );
}
