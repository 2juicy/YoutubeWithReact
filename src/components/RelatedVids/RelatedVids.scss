.related-vids {
  & > * {
    margin-top: 4px;
  }
}
