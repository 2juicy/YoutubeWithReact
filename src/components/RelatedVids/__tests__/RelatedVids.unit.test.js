import React from "react";
import { shallow } from "enzyme";
import RelatedVids from "../RelatedVids";

describe("RelatedVids", () => {
  test("renders", () => {
    const wrapper = shallow(<RelatedVids />);
    expect(wrapper).toMatchSnapshot();
  });
});
