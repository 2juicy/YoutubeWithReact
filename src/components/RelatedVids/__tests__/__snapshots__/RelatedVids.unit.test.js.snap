// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RelatedVids renders 1`] = `
<div
  className="related-vids"
>
  <NextVid />
  <VidPreview
    horizontal={true}
  />
  <VidPreview
    horizontal={true}
  />
  <VidPreview
    horizontal={true}
  />
</div>
`;
